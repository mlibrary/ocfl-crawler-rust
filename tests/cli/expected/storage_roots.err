tests/cli/1.2 is not a storage root
