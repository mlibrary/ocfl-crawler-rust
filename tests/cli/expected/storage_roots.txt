storage root: tests/cli/1.0
storage root: tests/cli/1.1
tests/cli/1.2 is not a storage root